conf/Makefile.chibios

```diff
@@ -20,66 +19,77 @@
 # Boston, MA 02111-1307, USA.
 #
 
-
-# Verify that libopencm3 is compiled with chibios compatible float abi
-# correct abi is -mfloat-abi=softfp
-# on non chibios branches, libopencm3 is compiled with more optimised -mfloat-abi=hard
-TAG_ABI_VFP_COUNT= \
-	$(shell (readelf -A  $(PAPARAZZI_SRC)/sw/ext/libopencm3/lib/libopencm3_stm32f4.a | \
-	grep -c Tag_ABI_VFP_args))
-
-ifneq ($(TAG_ABI_VFP_COUNT),0)
-  $(error libopencm3_stm32f4.a is generated with chibios incompatible -mfloat-abi=hard, \
-	please recompile with -mfloat-abi=softfp))
-endif
-
-
 #
 # This is the common Makefile for target using chibios
+#
+# chibios directory
+CHIBIOS =  $(PAPARAZZI_SRC)/sw/ext/chibios
+
+# directory with board defines for chibios platforms (board specific)
+BOARD_DIR ?= $(BOARD)/chibios
```

Comment:
here you set BOARD_DIR to $(BOARD)/chibios, but for apogee you use `BOARD_DIR=$(BOARD)/v$(BOARD_VERSION)`...
Maybe we should stick to one schema?